nncf/experimental/onnx/statistics/collectors.py

```diff
@@ -89,3 +99,14 @@ def _register_input(self, x: ONNXNNCFTensor):
 
     def _get_statistics(self) -> ONNXMinMaxTensorStatistic:
         return ONNXMinMaxTensorStatistic(self._min_aggregate().tensor, self._max_aggregate().tensor)
+
+class ONNXMeanStatisticCollector(MeanStatisticCollector):
```

Comment:
Is it possible to create a common `MeanStatisticCollector`?

Reply:
It was created similarly with `TF...StatisticCollector`, `PT...StatisticCollector` and etc. Not sure that the common way is applicable here.